rules-threat-hunting/windows/powershell/powershell_script/posh_ps_win_api_library_access.yml

```diff
@@ -1,19 +1,23 @@
-title: Potential WinAPI Calls Via PowerShell Scripts
-id: 03d83090-8cba-44a0-b02f-0b756a050306
+title: WinAPI Function Calls Via PowerShell Scripts
+id: 9f22ccd5-a435-453b-af96-bf99cbb594d4
 related:
+    - id: ba3f5c1b-6272-4119-9dbd-0bc8d21c2702
+      type: similar
     - id: 03d83090-8cba-44a0-b02f-0b756a050306
       type: similar
+    - id: 19d65a1c-8540-4140-8062-8eb00db0bba5
+      type: similar
 status: experimental
-description: Detects use of WinAPI Functions in PowerShell scripts
+description: Detects calls to WinAPI functions from PowerShell scripts. Attackers can often leverage these API to avoid detection based on typical PowerShell function calls. Use this rule as a basis to hunt for interesting scripts.
```

Comment:
```suggestion
description: Detects calls to WinAPI functions from PowerShell scripts. Attackers can often leverage these APIs to avoid detection based on typical PowerShell function calls. Use this rule as a basis to hunt for interesting scripts.
```

Reply:
the description suggest this is a hunting rule. But it's in the normal rules directory?

Reply:
It is in the hunting folder